examples/nodejs/cache/package-lock.json

```diff
@@ -122,6 +122,7 @@
       "version": "0.9.5",
```

Comment:
why did this file change? just curious

Reply:
While I was experimenting with the ci build failures, I reinstalled my node modules